SoftLayer/CLI/dedicatedhost/cancel_guests.py

```diff
@@ -0,0 +1,34 @@
+"""Cancel a dedicated host."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import exceptions
+from SoftLayer.CLI import formatting
+from SoftLayer.CLI import helpers
+
+
+@click.command()
+@click.argument('identifier')
+@environment.pass_env
+def cli(env, identifier):
+    """Cancel all virtual guests of the dedicated host immediately.
+
+       Use the 'slcli vs cancel' command to cancel an specific guest
+    """
+
+    dh_mgr = SoftLayer.DedicatedHostManager(env.client)
+
+    host_id = helpers.resolve_id(dh_mgr.resolve_ids, identifier, 'dedicated host')
+
+    if not (env.skip_confirmations or formatting.no_going_back(host_id)):
+        raise exceptions.CLIAbort('Aborted')
+
+    result = dh_mgr.cancel_guests(host_id)
```

Comment:
After updated the manager function, just add a way to print out a nice table with the results of each guests' cancelation.

Reply:
Done. It prints a table like below

```
$ slcli --really dh cancel-guests 33333
:..........:...........................:..................................................................:
:    id    :        server name        :                              status                              :
:..........:...........................:..................................................................:
: 1111111  :    foobar.softlayer.com   :                            Cancelled                             :
: 2222222  :    wombat.softlayer.com   : Exception: A billing item is required to process a cancellation. :
:..........:...........................:..................................................................:

```